twine/__init__.py

```diff
@@ -24,7 +24,12 @@
 
 __copyright__ = "Copyright 2019 Donald Stufft and individual contributors"
 
-import importlib_metadata
+import sys
+
+if sys.version_info[:2] >= (3, 10):
+    from importlib import metadata as importlib_metadata
```

Comment:
This is different from below and makes grep'ping for the standard library harder for the next time we have to rely only one the backport library. Please be consistent here so in a few months we can more easily revert these changes. (I'm guessing here based on the patterns we've seen.)

Reply:
I would suggest pushing the boundary to (3, 11) instead of reverting. As I understand it, `importlib_metadata` is always meant to be there temporarily, until stdlib catches up.
(But of course, I'm only suggesting – it's your project!)